testing/tests/DevExpress.ui.widgets/diagramParts/toolbox.tests.js

```diff
@@ -27,24 +27,24 @@ QUnit.module('Toolbox', {
 }, () => {
     test('should render if toolbox.visibility is "visible"', function(assert) {
         this.instance.option('toolbox.visibility', 'visible');
-        const $accordion = $('body').find(Consts.TOOLBOX_ACCORDION_SELECTOR);
+        const $accordion = $('#qunit-fixture').find(Consts.TOOLBOX_ACCORDION_SELECTOR);
```

Comment:
why cannot we use body here?

Reply:
because we do not have the `body` element inside the shadow root